app/uk/gov/hmrc/apiplatformmicroservice/config/Scheduler.scala

```diff
@@ -51,5 +53,9 @@ class Scheduler @Inject()(deleteUnverifiedDevelopersJobConfig: DeleteUnverifiedD
     Seq.empty
   }
 
-  override lazy val scheduledJobs: Seq[ScheduledJob] = deleteUnverifiedDevsJob ++ deleteUnregisteredDevsJob
+  lazy val unusedApplicationJobs: Seq[ScheduledJob] =
+    Seq(updateUnusedSandboxApplicationRecordsJob, updateUnusedProductionApplicationRecordJob)
+      .filter(_.isEnabled)
```

Comment:
I like this. Maybe we could do it for all the jobs. A lot simpler than what we usually do.

Reply:
:+1:

Reply:
Yeah, seems a bit weird that we have to inject both the job itself and it's config into `Scheduler`, especially when that config is injected into the job...